src/screens/OtherProfile/index.js

```diff
@@ -229,7 +237,7 @@ const OtherProfile = () => {
         <View style={styles.wrapFollower}>
           <TouchableOpacity onPress={handleOpenFollowerUser} style={styles.wrapRow}>
             <React.Fragment>
-              <Text style={styles.textTotal}>{dataMain.follower_symbol}</Text>
+              <Text style={styles.textTotal(isAnonimity)}>{dataMain.follower_symbol}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change here involves passing `isAnonimity` as an argument to the `styles.textTotal` function. This allows the text color to be dynamically set based on whether the user is anonymous or not. It's a good practice to make the UI adaptable to different states.



<!-- This is an auto-generated comment by OSS CodeRabbit -->